docs/source/library_reference.rst

```diff
@@ -23,40 +23,70 @@ Python Test Runner
 .. warning::
     Python runners and associated APIs are an experimental feature and subject to change.
 
-.. automodule:: cocotb_tools.runner
-    :members:
-    :member-order: bysource
+.. currentmodule:: cocotb_tools.runner
```

Comment:
Another way I just learned about is starting the ref content with a `.`, e.g. ``:meth:`.Runner.build` ``. It may or may not help, haven't tried. I was gonna do that this morning to fix the broken links in the runner docs.

Reply:
I can't find documentation on the dot prefix. What does it do?

Reply:
Yeah it's hidden...

Reply:
That fixed it locally. Let's see.

Reply:
Looks like it works.